[

  {
    "name": "FileNotFoundError",
    "defaultMessage": "The requested file could not be found",
    "defaultExplanation": "The file could not be found",
    "defaultResponse": "Verify the file exists and retry the operation",
    "code": 601
  },
  {
    "name": "FolderNotFoundError",
    "defaultMessage": "The requested folder could not be found",
    "defaultExplanation": "The folder could not be found",
    "defaultResponse": "Verify the folder exists and retry the operation",
    "code": 602
  },
  {
    "name": "FolderRemoveError",
    "defaultMessage": "The requested folder could not be deleted",
    "defaultExplanation": "The folder could not be found",
    "defaultResponse": "Verify the folder exists and retry the operation",
    "code": 603
  },
  {
    "name": "FolderHasError",
    "defaultMessage": "The requested folder could not be deleted",
    "defaultExplanation": "The folder contains files or folders",
    "defaultResponse": "Please delete all files or folders within",
    "code": 604
  },
  {
    "name": "UploadHasError",
    "defaultMessage": "The upload operation failed with errors.",
    "defaultExplanation": "An error occured during the file upload.",
    "defaultResponse": "Error has been logged and Admin notified",
    "code": 605
  },
  {
    "name": "OperationFailed",
    "defaultMessage": "The operation failed with errors.",
    "defaultExplanation": "An error occured during the file operation.",
    "defaultResponse": "Error has been logged and Admin notified",
    "code": 606
  },
  {
    "name": "OperationHasErrors",
    "defaultMessage": "The operation completed with errors.",
    "defaultExplanation": "Errors occured during this file operation.",
    "defaultResponse": "Errors have been logged and Admin notified",
    "code": 607
  },
  {
    "name": "ClientIdError",
    "defaultMessage": "This action is not allowed. We can not understand your request.",
    "defaultExplanation": "An internal server error occurred",
    "defaultResponse": "Error has been logged and Admin notified",
    "code": 608
  },
  {
    "name": "S3UploadHasError",
    "defaultMessage": "The upload operation failed with errors.",
    "defaultExplanation": "An error occured during the AWS S3 file upload.",
    "defaultResponse": "Error has been logged and Admin notified",
    "code": 609
  },
  {
    "name": "CriticalServerError",
    "defaultMessage": "iXit could not complete this request because it had a major internal error.",
    "defaultExplanation": "Rarely things go wrong here. We are probably aware of this and will have it fixed as soon as possible.",
    "defaultResponse": "Error has been logged and Admin notified",
    "code": 610
  }

]